lib/player.js

```diff
@@ -5485,6 +5486,51 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
         shaka.Player.EventName.AbrStatusChanged, data));
   }
 
+  /**
+   * @param {!shaka.util.Error} error
+   * @return {boolean}
+   * @private
+   */
+  canRecoverFromError_(error) {
```

Comment:
This method seems to be poorly-named. It sounds like it's a non-mutating function that returns whether the error can be recovered from, but the method actually attempts to recover.
I'd call it `tryToRecoverFromError_` instead. And maybe give it a description in the JSDoc?

Reply:
Renamed, thx